SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduler.java

```diff
@@ -471,7 +471,9 @@ public void handleCompletedTask(Optional<SingularityTask> task, SingularityTaskI
       stateCache.getActiveTaskIds().remove(taskId);
     }
 
-    taskManager.createLBCleanupTask(taskId);
+    if (task.isPresent() && task.get().getTaskRequest().getRequest().isLoadBalanced()) {
+      taskManager.createLBCleanupTask(taskId);
+    }
```

Comment:
This was to try and cut down on non-load-balanced tasks in the lb cleanup queue, especially since we are now exposing more of that data in the ui. Might be a good idea to still add it in the case where the task isn't present, I'll fix that.